components/pages/PastVotes.tsx

```diff
@@ -0,0 +1,68 @@
+import {
+  Banner,
+  Layout,
+  LoadingSpinner,
+  PageInnerWrapper,
+  PageOuterWrapper,
+  Pagination,
+  VotesTable,
+  VotesTableHeadings,
+  VotesTableRow,
+  VoteTimeline,
+} from "components";
+import { usePanelContext, useVotesContext, useVoteTimingContext } from "hooks";
+import styled from "styled-components";
+import { LoadingSpinnerWrapper } from "./styles";
+
+export function PastVotes() {
```

Comment:
This component is similar to the `Votes` component, but it shows past votes only.